### DISCUSSION TITLE
Apertura cancello in automatico quando si e vicini a questo

### DISCUSSION THREAD

**Discussion by ValerioGitH** [indomus/forum#2449] [Nuove idee, proposte, progetti e guide]
Salve, vorrei aprire il cancello automatico di casa quando sono vicino a questo, la rilevazione gps la trovo troppo "dispersiva" (nel senso che prende in considerazioni spazi troppo grandi così almento per uscire di casa sono costretto a fare manualmente. Avevo pensato al bluethoot, quindi usare uno shelly 1 plus o semplicemente un beans bluethoot che potrebbero rilevare la prossimità del mio smarthphone quando questo si trova a pochi metri. Cosa ne pensate? E' fattibile?

**Comment by 50494554524F** [comment]
ci stavo pensando anche io tempo fa a titolo di esercizio, ti butto giù quello che avevo partorito :D

quando rientri puoi usare il gps visto che il cancello ci metterà un po' ad aprirsi, siamo in sintonia

quando esci puoi controllare qualche livello di segnale, a me non piace perché poco affidabile il valore rilevato e qualche pacchetto può sempre arrivare o non arrivare se c'è interferenza radio nel punto di interesse... cmq fattibile

sono arrivato alla soluzione di strisciare un tag nfc o usare un sensore di movimento posizionato dove non passi usualmente, il cancello si aprirà mentre ti sistemi in macchina o arrivi

per non mettere mano all'impianto avrei usato un altro telecomando con bot

**Reply by ValerioGitH** [reply]
quindi utilizzare il segnale bluethoot pensi non sia fattibile?

**Reply by 50494554524F** [reply]
> quindi utilizzare il segnale bluethoot pensi non sia fattibile?

se sei certissimo che dove è il ricevitore bluetooth il segnale è sempre zero è ovviamente fattibile... altrimenti ti faccio un esempio: sto provando nuove cose ed ho un sensore finestre bluetooth vicino al pc, segnale -85db... come fai a capire se ti stai avvicinando o allontanando se già a 10cm è -85?!
se i dispositivi già ce li hai puoi provare, ma per non sono valori affidabili per farci qualcosa...

**Reply by bubez81** [reply]
io uso questo sistema con discreta soddisfazione https://github.com/dalehumby/ESPHome-Apple-Watch-detection

**Reply by ValerioGitH** [reply]
si può usare uno smarthphone invece che l'apple watch?
